### PR TITLE
PODS-9399 - Update method for getting tax year

### PR DESCRIPTION
**Pull request by LauraToddCodes**

### CODE REVIEW

File: app/controllers/DeclarationController.scala [hunk 1]
```diff
@@ -59,7 +59,11 @@ class DeclarationController @Inject()(
 
   def onPageLoad(waypoints: Waypoints): Action[AnyContent] = (identify andThen getData() andThen requireData) {
     implicit request =>
-      isReportDuplicate(request.pstr, 2023)
+      val currentTaxYear = request.userAnswers.get(TaxYearPage) match {
+        case Some(value) => value
+        case _ => TaxYear("2022")
```
Comment (jamdade-sharad) [inline app/controllers/DeclarationController.scala:64]: which tax year needed here? 2022 or 2023

**Review by jamdade-sharad — CHANGES_REQUESTED** [review]
which tax year needed here? 2022 or 2023

File: app/controllers/DeclarationController.scala (hunk 1 above)
Comment (LauraToddCodes) [reply app/controllers/DeclarationController.scala:64]: It doesn't really matter. It's just for me to get some info in QA from the logging.

**Review by jamdade-sharad — APPROVED** [review]
lgtm

**Pull request merged**